ross/api_report

```diff
@@ -0,0 +1,538 @@
+import numpy as np
+from scipy.signal import argrelextrema
+
+from rotor_assembly import Rotor
+import ross as rs
+
+import bokeh.palettes as bp
+from bokeh.plotting import figure
+from bokeh.models import (
+        ColumnDataSource,
+        HoverTool,
+        Span,
+        Label,
+)
+
+# set bokeh palette of colors
+bokeh_colors = bp.RdGy[11]
+
+
+class Report:
+    def __init__(self, rotor, safety_factor=1.0):
+        """Report according to standard analysis.
+
+        - Perform Stability_level1 analysis
+        - Apply Level 1 Screening Criteria
+        - Perform Stability_level2 analysis
+
+        Parameters
+        ----------
+        rotor : object
+            A rotor built from rotor_assembly
+        """
+        self.rotor = rotor
+        self.safety_factor = safety_factor
+
+    @classmethod
+    def from_saved_rotors(cls, path):
+        rotor = rs.Rotor.load(path)
+        return cls(rotor)
+
+    def static_forces(self, maxspeed, mode):
```

Comment:
At the end, the primary purpose of this function is to calculate the unbalance forces, so I think we should change the name of the function to something like `.calculate_unbalance_forces()`. Another option is that we split this function in two, the first would be `.static_forces()` and it would return the `Fb` array, and the second would be calculate_unbalance_forces()` and it would use the `Fb` array as argument.

Reply:
I did two separated methods:
- one to get the reaction forces
- other to calculate the unbalance forces